README.md

```diff
@@ -1,94 +1,274 @@
-- [`./install`](./install/) - collection of things to support the various installation, signing and notarization requirements.
-- [`./docs`](./docs) Some small content to be inserted into mac pgk file
+```
+tools-lukso-cli
+│
+└───abis                  // Binary Interfaces of LUKSO Smart Contracts
+│
+└───cmd                   // Commands
+│   └───lukso             // LUKSO CLI
+│
+└───contracts             // Solidity Contracts
+│   └───bindings          // Bindings generated from ABIs
+|
+└───docs                  // Updates for Installation Progress
+|
+└───install               // Mandatory Installation Tools
+│   └───cf-wrangler       // Manager for Cloudflare Workers
+│   └───docs-processor    // Markdown to Page Converter
+│   └───macos_packages    // MacOS Codesigning Scripts
+|
+└───pid                   // Process ID Management
+```
+
+## CLI Installation Script
 
-## CLI Installation
 ```sh
-curl https://install.lukso.network | sh
+https://install.lukso.network
 ```
 
-## Downloading and Installing LUKSO
+**Running this script will install the full LUKSO CLI Tool on Mac and Linux.**
+**Installation directory: `usr/local/bin/lukso`**
 
-```bash
-# 1. To install the LUKSO Command Line Interface (CLI)
-$ curl https://install.lukso.network | sh
+## Node Folder Structure
 
-# 2. Create a working folder where you want your clients to store their data
-$ mkdir myLUKSOFolder && cd ./myLUKSOFolder
+> Initializing a LUKSO node will not overwrite existing config, data or keystore folders
 
-# 3. This will initialize your working folder by downloading all network configs from
-# https://github.com/lukso-network/network-configs
-# NOTE: This will not overwrite any existing config, data or keystore folders
-$ lukso init
+```
+lukso-node
+│
+└───configs               // Blockchain Configuration
+|   | config.yaml         // Network Configuration
+|   | genesis.json        // Genesis JSON Data
+|   | genesis.ssz         // Genesis Validator File
+│
+└───data                  // Blockchain Data Storage
+│   └───consensus_data    // Storage of Consensus Client
+│   └───execution_data    // Storage of Execution Client
+│   └───validator_data    // Storage of Validator Client
+│
+└───keystore              // Validator Secrets and Keys
+│   └───keys              // Encrypted Private Keys
+│   └───...               // Folders & Files for Signature Creation
+|   | pubkeys.json        // Validator Public Keys
+|
+└───transaction_wallet    // Validators Transaction Wallet
+|
+| deposit_data.json       // Deposit JSON for Validators
+| node_config.yaml        // Node Configuration File
+```
 
-# 4. Install your clients. It will ask you which ones you want to install
-$ lukso install
+## External Sources
+
+- The network configuration is fetched from [lukso-network/network-configs](https://github.com/lukso-network/network-configs)
+
+- Deposit key can be generated using [lukso-network/tools-key-gen-cli](https://github.com/lukso-network/tools-key-gen-cli)
```

Comment:
We can also mention the wagyu client: https://github.com/lukso-network/tools-wagyu-key-gen